### PR TITLE
fix: Fixing typo. It's postgres, not postres.

### PR DESCRIPTION
**Pull request by philippe-lavoie**
Quick fix for a typo on the word postgres.

### CODE REVIEW

File: activity/sqlquery/descriptor.json [hunk 1]
```diff
@@ -4,15 +4,15 @@
 	"version": "0.9.0",
 	"title": "SQL Database Activity",
   	"description": "Flogo Activity for SQL",
-  	"homepage": "https://github.com/project-flogo/contrib/tree/master/activity/sqlquery",
+  	"homepage": "https://github.com/philippe-lavoie/contrib/tree/master/activity/sqlquery",
```
Comment (mellistibco) [inline activity/sqlquery/descriptor.json]:
```suggestion
  	"homepage": "https://github.com/project-flogo/contrib/tree/master/activity/sqlquery",
```

File: activity/sqlquery/go.mod [hunk 2]
```diff
@@ -1,4 +1,4 @@
-module github.com/project-flogo/contrib/activity/sqlquery
+module github.com/philippe-lavoie/contrib/activity/sqlquery
```
Comment (mellistibco) [inline activity/sqlquery/go.mod]:
```suggestion
module github.com/project-flogo/contrib/activity/sqlquery
```

Comment (philippe-lavoie) [reply activity/sqlquery/go.mod]: Done

File: activity/sqlquery/go.mod [hunk 3]
```diff
@@ -3,6 +3,7 @@ module github.com/project-flogo/contrib/activity/sqlquery
 go 1.12
 
 require (
+	github.com/project-flogo/contrib/activity/sqlquery v0.0.0-20191205162113-393b3f6aa926
```
Comment (mellistibco) [inline activity/sqlquery/go.mod]: this shouldn't be here

**Pull request merged**